core_backend/add_new_data_to_db.py

```diff
@@ -0,0 +1,412 @@
+import argparse
+import json
+import random
+from collections import defaultdict
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from datetime import datetime, timedelta
+
+import pandas as pd
+import urllib3
+from add_users_to_db import ADMIN_API_KEY
+from app.config import (
+    LITELLM_API_KEY,
+    LITELLM_ENDPOINT,
+    LITELLM_MODEL_DEFAULT,
+)
+from app.database import get_session
+from app.question_answer.models import ContentFeedbackDB, QueryDB, ResponseFeedbackDB
+from app.urgency_detection.models import UrgencyQueryDB
+from app.users.models import UserDB
+from app.utils import get_key_hash
+from litellm import completion
+from sqlalchemy import (
+    select,
+)
+from urllib3.exceptions import InsecureRequestWarning
+
+# To disable InsecureRequestWarning
+urllib3.disable_warnings(InsecureRequestWarning)
+
+try:
+    import requests  # type: ignore
+except ImportError:
+    print(
+        "Please install requests library using `pip install requests` "
+        "to run this script."
+    )
+
+MODELS = [
+    (QueryDB, "query_datetime_utc"),
+    (ResponseFeedbackDB, "feedback_datetime_utc"),
+    (ContentFeedbackDB, "feedback_datetime_utc"),
+    (UrgencyQueryDB, "message_datetime_utc"),
+]
+
+parser = argparse.ArgumentParser(
+    description="Bulk add data to the database.",
+    usage="""
+    python add_content_to_db.py [-h] --csv CSV --domain DOMAIN [--language LANGUAGE]
+
+    (example)
+    python add_new_dummy_data.py \\
+        --csv generated_questions.csv \\
+        --host localhost:8000 \\
```

Comment:
```suggestion
        --domain localhost:8000 \
```
We can make all the `\\`'s into `\`?